### ISSUE TITLE
backtick strings are reported as syntax errors

### ISSUE THREAD

**Issue by mooow** [dude719/JetBrains-NASM-Language#18]
I have defined strings like this
```nasm
string: db `Hello from assembly\n\0`
```
which, according to the docs, is perfectly fine (and in fact, it compiles with no problems).

However, the tools reports them as a syntax error: 
```
Error:(22, 12) <numeric expr>, NASMTokenType.+, NASMTokenType.-, NASMTokenType.STRING or NASMTokenType.~ expected, got '`'
```

Of course I can redefine them using normal quotes, but I don't like doing so.

**Comment by ajkhoury** [comment]
I'll add a rule for backtick strings. Thanks for opening an issue for this.

**Comment by ajkhoury** [comment]
Fixed in 12441e94e43c5222ec67c179577d4a4cfb2b143b
